pkg/operator/ceph/object/controller.go

```diff
@@ -483,3 +490,24 @@ func (r *ReconcileCephObjectStore) startMonitoring(objectstore *cephv1.CephObjec
 	logger.Info("starting rgw healthcheck")
 	go rgwChecker.checkObjectStore(r.objectStoreChannels[objectstore.Name].stopChan)
 }
+
+func (r *ReconcileCephObjectStore) verifyObjectUserCleanup(objectstore *cephv1.CephObjectStore) (reconcile.Result, bool) {
+	cephObjectUsers, err := r.context.RookClientset.CephV1().CephObjectStoreUsers(objectstore.Namespace).List(metav1.ListOptions{})
```

Comment:
Nit, feel free to take it or not but with controller runtime, I believe it's a good habit to start using `client.Client` instead of the `RookClientset`, feature-wise it's the same though.

Reply:
@leseb Okay. But I was not able to `CephObjectStoreUsers(objectstore.Namespace).List(metav1.ListOptions{})` api from `r.client.`
Am I looking in the wrong places?